recipes/glow/meta.yaml

```diff
@@ -0,0 +1,58 @@
+{% set name = "glow" %}
+{% set pypi_name = "glow.py" %}
+{% set version = "0.6.0.post1" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://test.pypi.io/packages/source/{{ pypi_name[0] }}/{{ pypi_name }}/{{ pypi_name }}-{{ version }}.tar.gz
+  sha256: 32736e84c2c3df8f62bf5afaaebe6d96f09228bb27f2d953d2b736de7a9a082d
+
+build:
+  noarch: python
+  number: 0
+  script: "{{ PYTHON }} -m pip install . -vv"
+
+requirements:
+  host:
+    - python
+    - pip
+  run:
+    - python >=3.7
+    - pyspark >=2.4.0
```

Comment:
We don't completely match https://github.com/projectglow/glow/blob/1aa2bfd45a043b2c2b9b7d59db09c834ab922dcd/python/setup.py#L23 here.